Cargo.toml

```diff
@@ -47,6 +48,8 @@ risc0-core = { version = "0.17.0", default-features = false, path = "risc0/core"
 risc0-sys = { version = "0.17.0", default-features = false, path = "risc0/sys" }
 risc0-zkp = { version = "0.17.0", default-features = false, path = "risc0/zkp" }
 risc0-zkvm = { version = "0.17.0", default-features = false, path = "risc0/zkvm" }
+risc0-zkvm-fault = { version = "0.17.0", default-features = false, path = "risc0/zkvm/fault" }
```

Comment:
Unfortunately, we can't actually publish the `risc0-zkvm-fault` crate, so we're going to have to get more clever about this. Perhaps we need to use bootstrap command to generate the entire ELF and ID and embed it as a source file inside `risc0-zkvm`.